STYLE-GUIDE.md

```diff
@@ -0,0 +1,23 @@
+# Exercism's Style Guide
+
+This document acts as a Style Guide for the language and wording used in exercises.
+
+## General principle
+
+All content should be written in US English.
+All units of measurement must be [SI](https://en.wikipedia.org/wiki/International_System_of_Units) or [SI-derived](https://en.wikipedia.org/wiki/SI_derived_unit) units.
+In the future other translations may occur, but the "official" Exercism language is US English.
+
+## Explain or substitute mathematical and esoteric terms.
```

Comment:
Should there be a similar rule for abbreviations, acronyms and initialisms ?

For example, a rule that is often used in academic writing is that an initialism should have its expansion in brackets immediately following its first use and thereafter can be used freely.

Reply:
I think we should have a no jargon rule. And don't use initialisms etc at all unless they are better known by the initialism (e.g. HTTP, HTML, radar) or if the text would be ridiculously verbose otherwise.

Reply:
In favour of no jargon. 
However, outside of the specific language related initialism, should we assume that all users will have knowledge of what I or you might consider better known ones, even for non-native US English speakers?

Yet we would still need to use initialisms in some cases. If we, hypothetically, agree that RADAR is jargon to non-US English speakers, then we would not have to be forced to use the long form each and every time it is mentioned, especially if it might form the name of test cases or some other oft repeated part of an exercise.

thus we need some kind of middle solution (hence my suggestion 😉).

I can think of a host of these that might be used depending on the exercise and track: FIFO, NASA, ABS, GIF, DIY, SONAR, SUV, UFO, CRUD, ANSI, API, ASCII, ICANN, CMYK, HTTP, FTP, DPI, DSL, JPEG, ... these are not especially unusual in programming or jargon-y, yet I can easily say that not everyone around the world would be familiar with all them

Reply:
Let's have a seperate PR and standalone discussion for that. I've created an issue 🙂 

https://github.com/exercism/problem-specifications/issues/1716